ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/channels/WindowedSeekableByteChannel.java

```diff
@@ -124,7 +118,13 @@ public long size() throws IOException {
     public SeekableByteChannel truncate(final long size) throws IOException {
         throw new TruncateNotAllowedException();
     }
-    
+
+    private void checkClosed() {
```

Comment:
could you pull this "closable" functionality to the parent class, and overload here if necessary?